openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-scriptrunner/src/tools/ScriptRunner/ScriptRunner.vue

```diff
@@ -1638,6 +1646,25 @@ export default {
     step() {
       Api.post(`/script-api/running-script/${this.scriptId}/step`)
     },
+    // This is called by processLine no matter the current state
+    handleWaiting() {
+      // First check if we're not waiting and if so clear the interval
+      if (this.state !== 'waiting' && this.state !== 'paused') {
+        this.clearWaiting()
+      } else if (this.waitingInterval !== null) {
+        // If we're waiting and the interval is active then nothing to do
+        return
+      }
+      // Create an interval to count every second
+      this.waitingInterval = setInterval(() => {
+        this.waitingTime += 1
```

Comment:
This won't be very accurate over long durations.  Should save start of waiting. And then calculate as now - starttime.